java/com/engflow/notificationqueue/README.md

```diff
@@ -0,0 +1,65 @@
+# Using the notification queue and event store APIs
+
+This documents explains how to make use of the **notification queue API** to
+obtain the notifications for a given build and use the notification data
+to obtain invocation events from the **event store API**. To do so,
+the [EngFlow API](https://github.com/EngFlow/engflowapis) is used as external
+dependency (see [WORKSPACE](../../../../WORKSPACE)).
```

Comment:
Maybe a brief explanation of the purpose of these APIs? I don't know much about them. What kinds of things might we be notified for? What is the event store?

Reply:
I think the description should be in the engflowapis repository as developers really don't need the example repo to use the APIs. I guess all API should be documented there in deep.

Reply:
@Yannic WDYT?